packages/telestion-client-types/src/json-serializable.ts

```diff
@@ -5,8 +5,9 @@
  * @see {@link JSON.stringify}
  */
 export type JsonSerializable =
+	| null
 	| number
 	| string
 	| boolean
-	| { [key: string]: JsonSerializable }
-	| Array<JsonSerializable>;
+	| { [key: string]: JsonSerializable | null | undefined }
+	| Array<JsonSerializable | null>;
```

Comment:
`JsonSerializable` already covers `null`, doesn't it?

```suggestion
	| { [key: string]: JsonSerializable | undefined }
	| Array<JsonSerializable>;
```

Reply:
Of course!